pkg/autoscaler/aggregation/bucketing_test.go

```diff
@@ -133,6 +133,39 @@ func TestTimedFloat64BucketsManyReps(t *testing.T) {
 	}
 }
 
+func TestTimedFloat64BucketsWindowTotal(t *testing.T) {
+	now := time.Now()
+	buckets := NewTimedFloat64Buckets(5*time.Second, granularity)
+
+	for i := time.Duration(0); i < 5; i++ {
```

Comment:
nit: Cast to duration in the loop? This reads a little odd.

Reply:
Done.